core/shared/src/main/scala/zio/ZIO.scala

```diff
@@ -1692,7 +1692,18 @@ sealed trait ZIO[-R, +E, +A] extends Serializable { self =>
     self &&& that
 }
 
-private[zio] trait ZIOFunctions extends Serializable {
+private[zio] trait ZIOIterableFunctions extends Serializable {
+  def partitionMap[A, A1, A2](iterable: Iterable[A])(f: A => Either[A1, A2]): (List[A1], List[A2]) =
```

Comment:
This needs to be `private[zio]` to avoid exposing it. I think I would probably just put the method in the ZIO companion object. Alternatively we could create a new `private[zio]` object in the `zio` package for this but I think it is probably not worth it if it is just going to contain a single method.